master/usage/index.md

```diff
@@ -1,5 +1,67 @@
----
-title: Using Calico
----
-
+---
+title: Using Calico
+---
+
 This section contains information on using Calico.
+
+## Calico software components
+
+There are several Calico components that may be ran as part of your
+installation, check out the
+[Getting Started]({{site.baseurl}}/{{page.version}}/getting-started)
+section for the components that are needed for your specific orchestrator.
+Here are the common components with links to relevant sections for configuring
+and operating those components.
+
+- [Calico node container]({{site.baseurl}}/{{page.version}}/reference/node/configuration)
+  - [Felix]({{site.baseurl}}/{{page.version}}/reference/node/reference/felix/configuration)
+  - [BGP]({{site.baseurl}}/{{page.version}}/usage/configuration/bgp)
+  - [Running Calico Node Container as a Service]({{site.baseurl}}/{{page.version}}/usage/configuration/as-service)
```

Comment:
I think all the places we say "Calico Node container" should really be "calico/node container".